src/VALIDATE.c

```diff
@@ -15,23 +15,23 @@
 int validate() {
     char *tempdesig[2]= {"employee","manager"};
     if (strcmp(e.name, "") == 0) {
-        gotoxy(0, 20);
+        gotoxy(0, 30);
         printf("Please enter valid employee name");
         getch();
         return 1;
     }
 
     /* Validate designation */
     if ((strcmp(e.desig, tempdesig[0]) != 0) && (strcmp(e.desig, tempdesig[1]) != 0)) {
-        gotoxy(0, 20);
+        gotoxy(0, 30);
         printf("Please enter valid designation!");
         getch();
         return 1;
     }
 
     /* Validate age */
     if ((e.age < 18) || (e.age > 120)) {

```

Comment:
Use macro define , do not hard code value

Reply:
This is been handled by our file_open function. It test the the file open open first after that marked logic is executed.

Reply:
This is normal age which need to be validated. So, it looks okay to me.